com.unity.render-pipelines.core/Editor/MaterialUpgrader.cs

```diff
@@ -57,6 +58,24 @@ class KeywordFloatRename
         }
         List<KeywordFloatRename> m_KeywordFloatRename = new List<KeywordFloatRename>();
 
+        internal enum RenameType
```

Comment:
Can be made public imo.

Reply:
Changed in 25554645a928746093a54734dbfc3f3143cd0208.